aspnet/security/authorization/limitingidentitybyscheme.rst

```diff
@@ -31,25 +31,24 @@ In this configuration two authentication middlewares have been added, one for co
 Selecting the scheme with the Authorize attribute
 -------------------------------------------------
 
-As no authentication middleware is configured to automatically run and create an identity you must, at the point of authorization choose which middleware will be used. The simplest way to select the middleware you wish to authorize wish is to use the ``AuthenticationSchemes`` parameter. This parameter accepts a comma delimited list of Authentication Schemes to use. For example;
-
+As no authentication middleware is configured to automatically run and create an identity you must, at the point of authorization choose which middleware will be used. The simplest way to select the middleware you wish to authorize wish is to use the :dn:prop:`~Microsoft.AspNetCore.Authorization.AuthorizeAttribute.ActiveAuthenticationSchemes` property. This property accepts a comma delimited list of Authentication Schemes to use. For example;
 .. code-block:: c#
 
- [Authorize(AuthenticationSchemes = "Cookie,Bearer")]
+ [Authorize(ActiveAuthenticationSchemes = "Cookie,Bearer")]
```

Comment:
ugh, wish I had renamed this property as well when I dropped the Active on the AuthorizePolicy